playground/mocks/data/idp/idx/okta-verify-uv-verify-enable-biometrics.json

```diff
@@ -135,7 +135,7 @@
       "type": "array",
       "value": [
         {
-          "message": "We received your response, but your organization requires biometrics. On a device that supports biometrics, enable biometrics for your account and try again.",
+          "message": "Your response was received, but your organization requires biometrics. Make sure you meet the following requirements, then try again:\\r\\n\\u2022 Your device supports biometrics\\r\\n\\u2022 Okta Verify is up-to-date\\r\\n\\u2022 In Okta Verify, biometrics are enabled for your account\\r\\n\\r\\n",
```

Comment:
oh... is this how the API error is returned? I don't think the API should return errors with line breaks...

Reply:
yea, this is the backend PR
https://github.com/okta/okta-core/pull/51982